pkg/controller/controller_utils.go

```diff
@@ -916,14 +930,20 @@ func AddOrUpdateTaintOnNode(c clientset.Interface, nodeName string, taint *v1.Ta
 // won't fail if target taint doesn't exist or has been removed.
 // If passed a node it'll check if there's anything to be done, if taint is not present it won't issue
 // any API calls.
-func RemoveTaintOffNode(c clientset.Interface, nodeName string, taint *v1.Taint, node *v1.Node) error {
+func RemoveTaintOffNode(c clientset.Interface, nodeName string, node *v1.Node, taints ...*v1.Taint) error {
+	if len(taints) == 0 {
+		return nil
+	}
 	// Short circuit for limiting amount of API calls.
 	if node != nil {
 		match := false
+	MATCHED:
```

Comment:
hm... try to add unit test when creating this PR; it will import `node/testutil` for `FakeNodeHandler`, although it did not introduce Circular Dependency. Do you think we should move it `node/testutil` to `controller/testutil` which is also used by cloud controller, or just import it for test.